UIDokan/Sources/UIDokan/DesignSystem/CustomFont.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  File.swift
+//  
+//
+//  Created by ziad on 05/07/2022.
+//
+
+import UIKit
+struct CustomFontDescriptor{
+    let font: Font
+    let size: CGFloat
+    let style: UIFont.TextStyle
+}
```

Comment:
- Should be added to a separate file named `FontDescriptor` for example. 
- Name is misleading since we already have [UIFontDescriptor](https://developer.apple.com/documentation/uikit/uifontdescriptor)